README.md

```diff
@@ -93,7 +93,7 @@ where you can substitube `DEBUG=1` for any environment variable you desire.
 
 Before the development server can be ran, the database must be running. This project is configured to use [PostgreSQL](https://www.postgresql.org/).
 
-You may install Postgres on your machine if you wish, but we recommend running it locally using docker. A docker-compose service is available in [development/docker-compose.yml](/home/graham/ieee/hackathon-template/README.md). To run the database:
+You may install Postgres on your machine if you wish, but we recommend running it locally using docker. A docker-compose service is available in [development/docker-compose.yml](what-is-this). To run the database:
```

Comment:
@albogdan the link in the hackathon-site repo is also broken. Any what idea this is or what to replace it with?

Reply:
uhh just remove the link and leave it as `development/docker-compose.yml` I think